### PR TITLE
Dynamically declare p2p connections

### PR DESCRIPTION
**Pull request by corsacca**

### CODE REVIEW

**Comment by corsacca** [comment]
Todo: is creating the "mirror" connection.

If on the contact post_type i create a connection field to groups.
We should have an option for that field to also show up on the groups page.

File: dt-core/admin/menu/tabs/tab-custom-fields.php [hunk 1]
```diff
@@ -881,6 +882,8 @@ private function add_field( $post_type ){
         global $wp_post_types;
         $post_type = sanitize_text_field( wp_unslash( $post_type ) );
         $tile_options = DT_Posts::get_post_tiles( $post_type );
+        global $wp_post_types;
```
Comment (micahmills) [inline dt-core/admin/menu/tabs/tab-custom-fields.php:885]: `global $wp_post_types`  is already declared a few lines above

**Review by micahmills — CHANGES_REQUESTED** [review]
Things seem to be working as expected on the individual post pages. The Bulk edit doesn't work for connection fields now that they are dynamic. There appears to be a JS error on page load that is stopping the bulk edit to complete.

File: dt-core/admin/menu/tabs/tab-custom-fields.php (hunk 1 above)
Comment (corsacca) [reply dt-core/admin/menu/tabs/tab-custom-fields.php:885]: Can't have that!

Comment (corsacca) [reply dt-core/admin/menu/tabs/tab-custom-fields.php]: Thanks for spotting it.

**Comment by corsacca** [comment]
> Things seem to be working as expected on the individual post pages. The Bulk edit doesn't work for connection fields now that they are dynamic. There appears to be a JS error on page load that is stopping the bulk edit to complete.

@micahmills, I fixed a bulk update api call issue: https://github.com/DiscipleTools/disciple-tools-theme/pull/1396/commits/c273ba5e753a637718c2a452ff3a897d0b4ac4d9
and had fixed another one earlier with: https://github.com/DiscipleTools/disciple-tools-theme/pull/1396/commits/8d4ca7ce81c2291c34b6f72284a7de65d6f54a69

**Comment by micahmills** [comment]
That seems to fix it. Good Work!

**Review by micahmills — APPROVED** [review]

**Pull request merged**